platform/core/src/services/DicomMetadataStore/DicomMetadataStore.ts

```diff
@@ -186,6 +186,13 @@ const BaseImplementation = {
       madeInClient,
     });
   },
+  updateSeriesMetadata(seriesMetadata) {
+    const { StudyInstanceUID, SeriesInstanceUID } = seriesMetadata;
+    const study = _getStudy(StudyInstanceUID);
+    if (study) {
+      study.setSeriesMetadata(SeriesInstanceUID, seriesMetadata);
```

Comment:
what about if there is no series with that UID

Reply:
Updated, thanks!